liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/LooseEarApplication.java

```diff
@@ -4,6 +4,7 @@
 import java.util.Set;
 
 import org.apache.maven.artifact.Artifact;
+import org.apache.maven.shared.mapping.MappingUtils;
```

Comment:
This file needs a copyright statement. Look at other files that would have been created around the same time to figure out the first year that should be listed.